templates/full-stack.template.yaml

```diff
@@ -253,28 +263,91 @@ Resources:
         - AccountStack
         - Outputs.PipelineArtifactBucket
 
-Outputs:
-  RepoCloneUrlSsh:
-    Value: !GetAtt 
-      - ProjectSetupStack
-      - Outputs.RepoCloneUrlSsh
-  RepoCloneUrlHttp:
-    Value: !GetAtt 
-      - ProjectSetupStack
-      - Outputs.RepoCloneUrlHttp
-  IAMUserName:
-    Value: !GetAtt 
-      - ProjectSetupStack
-      - Outputs.IAMUserName
-  GitUserName:
-    Description: Username for access to the CodeCommit repository
-    Value: !GetAtt 
-      - ProjectSetupStack
-      - Outputs.GitUserName
-  GitPassword:
-    Description: Password for access to the CodeCommit repository
-    Value: !GetAtt 
-      - ProjectSetupStack
-      - Outputs.GitPassword
+  PipelineArtifactBucketParameter:
```

Comment:
Creating parameters for custom data required by the dynamic pipeline creator lambda.